AsyncDisplayKit/ASImageNode.mm

```diff
@@ -186,13 +188,23 @@ - (void)dealloc
 - (CGSize)calculateSizeThatFits:(CGSize)constrainedSize
 {
   ASDN::MutexLocker l(__instanceLock__);
-  // if a preferredFrameSize is set, call the superclass to return that instead of using the image size.
-  if (CGSizeEqualToSize(self.preferredFrameSize, CGSizeZero) == NO)
-    return [super calculateSizeThatFits:constrainedSize];
-  else if (_image)
-    return _image.size;
-  else
-    return CGSizeZero;
+  
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+  // If a preferredFrameSize is set, call the superclass to return that instead of using the image size.
+  if (CGSizeEqualToSize(self.preferredFrameSize, CGSizeZero) == NO) {
+#if DEBUG
+    NSLog(@"Using -[ASDisplayNde preferredFrameSize] is deprecated.");
```

Comment:
I don’t think we should use `LOG()`. We would like to have the log message in development under all circumstances. And `LOG` is currently disabled in development and production